terraform/environments/mlra/modules/codebuild/main.tf

```diff
@@ -50,11 +63,12 @@ resource "aws_s3_bucket_versioning" "report_versioning" {
 ######################################################
 
 resource "aws_ecr_repository" "local-ecr" {
+  #checkov:skip=CKV_AWS_136:TODO Will be addressed as part of https://dsdmoj.atlassian.net/browse/LASB-3390
   name                 = "${var.app_name}-local-ecr"
-  image_tag_mutability = "MUTABLE"
+  image_tag_mutability = "IMMUTABLE"
```

Comment:
This is changing the way we push images to the image repo - this is changing the behaviour of the image releases - I'd suggest we keep this to another task outside of this PR and have a wider discussion with how we want to manage image versions in our image repo's?

Reply:
Fair point, reverted